lib/src/layer/polyline_layer/painter.dart

```diff
@@ -31,9 +31,8 @@ class _PolylinePainter<R extends Object> extends CustomPainter {
 
     for (final projectedPolyline in polylines.reversed) {
       final polyline = projectedPolyline.polyline as Polyline<R>;
-      if (polyline.hitValue == null) {
-        continue;
-      }
+
+      if (polyline.hitValue == null) continue;
```

Comment:
This is super nitty and I won't argue over it but I feel like the extra indent helps with highlighting the control flow :shrug:

If on the other hand the body was just incrementing a counter, I wouldn't care.

Reply:
Yeah, I know what you mean. I've reverted the commit anyways, and won't add this back.